README.md

```diff
@@ -86,6 +94,43 @@ import pybamm_cookie
 
 pybamm_cookie.__version__
 ```
+### Project Layout
+```bash
+.
+├── .github
+│   └── workflows
+├── docs
+│   └── _static
+├── src
+│   ├── pybamm_example_project
+│   ├── models
+│   │   └── input
+│   │  
+│   └─ parameters
+│       └── input
+│  
+├── tests
+│    ├── generated_project_tests
+│    │  
+│    └── user_tests
+├── noxfile.py
+└─ pyproject.toml
+```
+A basic generated project with `pybamm-cookie` has a directory structure like the one depicted above. Each directory has a special purpose and is highly recommended to use them for their specific purposes to avoid any errors.
+- `docs/` contains the basic configuration for building documentation using [sphinx](https://www.sphinx-doc.org/).
+- `src/` is wraps the main package files, under a directory with your specified project name. This is where you'd be writing all your code.
+    - `models/input/` - Contains the source code for battery models. Refer [here](https://docs.pybamm.org/en/latest/source/api/models/index.html) to learn how to write your own battery models.
+    - `parameters/input/` - Contains the source code for parameters sets. Refer [here](https://docs.pybamm.org/en/stable/source/api/parameters/parameter_sets.html) to learn more on parameter sets.
+- `tests/` The tests directory contains two sub-directories -
+    - `generated_project_tests/` - Contains pre-defined tests for verifying if the units inside generated project work well. It can be verified using `nox -s generated-project-tests` command.
+    - `user_tests/` - This directory contains all the user written tests, the example tests can be copied and modified as per your own needs. We suggest you use [pytest](https://docs.pytest.org/en/stable/) to write tests. The tests can be then run by executing `nox -s user-tests`.
+- `noxfile.py` contains basic `nox sessions` for building documentation and runnings tests.
+- `pyproject.toml` has project metadata and build backend configurations.
+- `github/workflows/` has all the basic CI configurations to automate tests and builds on [GitHub Actions](https://github.com/features/actions).
```

Comment:
It is not good practice to hyperlink the word "here" with URLs – it isn't very descriptive of what the link refers to and has negative accessibility implications.

More information is available below: 
1. https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/
2. https://www.accessguide.io/guide/descriptive-link-text